<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>시간단위</string>
       </property>
       <widget class="QRadioButton" name="radio_day">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>108</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>일봉</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_week">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>108</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>주봉</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_month">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>108</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>월봉</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>409</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>5일 이동평균선</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>409</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>20일 이동평균선</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>150</y>
          <width>43</width>
          <height>22</height>
         </rect>
        </property>
        <property name="singleStep">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>수량</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox3">
       <property name="text">
        <string>60일 이동평균선</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>611</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;종목코드&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mesage :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Click</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
